src/Symfony/Component/DependencyInjection/Attribute/AutoconfigureTag.php

```diff
@@ -19,8 +19,17 @@
 #[\Attribute(\Attribute::TARGET_CLASS | \Attribute::IS_REPEATABLE)]
 class AutoconfigureTag extends Autoconfigure
 {
-    public function __construct(string $name = null, array $attributes = [])
-    {
+    public function __construct(
+        /*
+         * The tag name to add.
+         */
+        string $name = null,
+
+        /*
+         * The tag attributes to attach to the tag.
```

Comment:
This should use `@param` on the method instead as those are not promoted properties. Otherwise, we won't have the benefits of exposing the description in IDEs supporting to show the phpdoc.